jboss-eap-clustered-vmss-rhel8/README.md

```diff
@@ -49,7 +49,8 @@ Following is the Architecture:
 
 To learn more about the JBoss Enterprise Application Platform, visit [Red Hat EAP 7.2 documentation](https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.2/).
 
-## Subscriptions and Costs - READ the instructions in their entirety before deploying.
+## Subscriptions and Costs
+READ the instructions in their entirety before deploying.
```

Comment:
We have completely changed the structure of “Subscriptions and Costs” section in the new Draft in https://github.com/SpektraSystems/redhat-mw-cloud-quickstart/blob/master/jboss-eap-standalone-rhel7/README.md. Please review and let us know the feedback, so that we can replicate the same format here